### ISSUE TITLE
Ajouter Type Generic

### ISSUE THREAD

**Issue by MinePlugins** [rezolv-fr/jeedoredaemon-dotnet#2]
Bonjour,

Par default le plugins n'ajoute pas le type generique de l'equiment, quand il y a beaucoup d'equipement cela fait beaucoup d'edition manuel.

Merci d'avance

**Comment by Heracles44** [comment]
Vous n'avez qu'à faire un vituel pour chaque volet et mettre vous me les types génériques. C'est ce que j'ai fait. Ainsi, je peux les remonter dans GHF sans problème :)

**Comment by MinePlugins** [comment]
Pas besoin de virtuel pour ca justement, il suffit d'aller sur chaque equiment et dans chaque command pour y associer le type generique, mais avec 50 equipement ca fais beaucoup de point and click qui pourrai etre pris de base dans le plugin

**Comment by JeedHome44** [comment]
oui mais le dev ne répond jamais ici... je ne sais pas pourquoi. j'essaie encore de mettre à jour son plugin pour que le démon soit enfin pris en charge automatiquement par jeedom mais il n'explique pas comment résoudre le problème.... pas top de faire payer un plugin 8 euros pour ensuite ne pas répondre aux personnes qui ont payées!

**Comment by MinePlugins** [comment]
J'ai arrêté d'utiliser jeedom et jeedore, maintenant je suis sous homeassistant pour avoir des dev qui réponde et qui font sans payer, d'ailleurs y a un Addon auxquelles j'ai participé pour gérer tydom dans hassio tydom2mqtt

**Comment by JeedHome44** [comment]
salut! cet addon sur hassio gère les volets? j'ai home assistant à côté de jeedom donc ca pourrait m'aider!

**Comment by MinePlugins** [comment]
Oui il gère volet, light, alarm pour le moment

**Comment by MinePlugins** [comment]
Oui il gère volet, light, alarm pour le moment

**Comment by JeedHome44** [comment]
c'est facile à mettre en place sur hassio? j'entends beaucoup parler de mqtt mais je ne sais pas ce que c'est

**Comment by JeedHome44** [comment]
j'ai que des volets sur tydom donc ca m'aiderait bien

**Comment by MinePlugins** [comment]
Oui, c'est assez facile, faut comprendre le principe de channel (topic) Sur lesquelle les info passe, et sinon si vous avez hassio avec supervisor il existe aussi sur mon GitHub in add-on auto qui configure tous seul les équipement

**Comment by JeedHome44** [comment]
oui j'ai Supervisor sur mon Hassio! comment faire svp pour mettre votre github automatique?

**Comment by JeedHome44** [comment]
j'ai aussi HACS

**Comment by MinePlugins** [comment]
En ajoutant ce repo si dans supervisor puis en installant le module tydom2mqtt nouvellement apparu https://github.com/MinePlugins/hassio_addons

**Comment by MinePlugins** [comment]
Biensur après il faut modifier la configuration pour renseigner les infos de tydom et les info du serveur matt (PS: il y a un Addon officiel qui gère le serveur mqtt)

**Comment by JeedHome44** [comment]
oula je ne comprends pas ce qu'est mqtt donc ca va être difficile pour moi

**Comment by MinePlugins** [comment]
C'est un moyen de communication pense pour l'IoT https://www.google.com/amp/s/www.lemagit.fr/conseil/Internet-des-Objets-bien-comprendre-MQTT%3famp=1

**Comment by JeedHome44** [comment]
Salut! 

J'ai mis MQTT broker sur Hassio, j'ai créé un user spécialement pour mqtt comme dit dans la doc de l'addon. Ensuite, j'ai installé l'intégration MQTT avec découverte automatique des nouveaux appareils.

J'ai installé par Supervisor Tydom2mqtt. Dans les configurations de ce dernier, j'ai renseigné les informations demandées.

Rien n'apparait sur MQTT. Est-ce normal? Comment as-tu fait?
Faut-il mettre la tydom dans un mot particulier?

**Comment by fusco** [comment]
> J'ai arrêté d'utiliser jeedom et jeedore, maintenant je suis sous homeassistant pour avoir des dev qui réponde et qui font sans payer, d'ailleurs y a un Addon auxquelles j'ai participé pour gérer tydom dans hassio tydom2mqtt

Je vais regarder de ce pas ! Car ça me saoule aussi de n’avoir aucune réponse !
